static/vendor/instantpage-5.1.0.js

```diff
@@ -1,2 +1,151 @@
 /*! instant.page v5.1.0 - (C) 2019-2020 Alexandre Dieulot - https://instant.page/license */
```

Comment:
Accidental addition belonging to separate PR perhaps?

Reply:
prettier... :D